src/ng2-datepicker/ng2-datepicker.component.ts

```diff
@@ -104,7 +105,7 @@ export const CALENDAR_VALUE_ACCESSOR: any = {
   providers: [CALENDAR_VALUE_ACCESSOR],
 })
 export class DatePickerComponent implements ControlValueAccessor, OnInit {
-  @Input() options: DatePickerOptions;
+  @Input() options: IDatePickerOptions;
```

Comment:
I had to change this, to prevent the situation, when the used did not set a property like "todayText" but wanted to use the standard local value.